librz/core/cmd/cmd.c

```diff
@@ -3989,32 +4011,21 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(arged_stmt) {
 	res = rz_cmd_call_parsed_args(state->core->rcmd, pr_args);
 	if (res == RZ_CMD_STATUS_WRONG_ARGS) {
 		const char *cmdname = rz_cmd_parsed_args_cmd(pr_args);
-		eprintf("Wrong number of arguments passed to `%s`, see its help with `%s?`\n", cmdname, cmdname);
-		RzCmdDesc *cd = rz_cmd_get_desc(state->core->rcmd, cmdname);
-		if (cd) {
-			char *cmdname_help = rz_str_newf("%s?", cmdname);
-			if (!cmdname_help) {
-				goto err;
-			}
-			RzCmdParsedArgs *help_pra = rz_cmd_parsed_args_newcmd(cmdname_help);
-			if (!help_pra) {
-				goto err;
-			}
-			bool use_color = state->core->print->flags & RZ_PRINT_FLAGS_COLOR;
-			char *help_msg = rz_cmd_get_help(state->core->rcmd, help_pra, use_color);
-			if (!help_msg) {
-				goto help_pra_err;
-			}
-			eprintf("%s", help_msg);
-			free(help_msg);
-		help_pra_err:
-			rz_cmd_parsed_args_free(help_pra);
-		}
+		eprintf("Wrong number of arguments passed to `%s`, see its help with `%s?`\n\n", cmdname, cmdname);
```

Comment:
do we really want to use eprintf ?

Reply:
For now, I think yes because it is used in other places here as well.